cfme/automate/service_dialogs.py

```diff
@@ -200,11 +180,8 @@ def element(self, element_data):
             '[contains(normalize-space(.), "{}")]/..'.format(element_data))
 
     def reorder_elements(self, tab, box, *element_data):
-        sel.force_navigate('service_dialog_edit', context={'dialog': self})
-        if version.current_version() > "5.5":
-            tree = accordion.tree("Dialog")
-        else:
-            tree = Tree("dialog_edit_treebox")
+        navigate_to(self, 'Edit')
+        tree = accordion.tree("Dialog")
         tree.click_path(self.label, tab, box)
```

Comment:
If the tree is not used anywhere else in the method, you can do `accordion.tree('Dialog', self.label, tab, box)` or something like that instead.